pkg/pillar/scripts/device-steps.sh

```diff
@@ -578,7 +587,10 @@ while true; do
         echo "$(date -Ins -u) NTP server changed from $NTPSERVER to $ns"
         NTPSERVER="$ns"
         kill "$(cat /run/ntpd.pid)"
+        echo "$(date -Ins -u) ntpd -q -n -g -p $NTPSERVER"
```

Comment:
Command and output mismatch for some reason here and few lines below

Reply:
Oops. Fixed to also pass -g to allow the initial jump for the case where pool.ntp.org didn't work.